feat: switch to new base version

weberval opened the pull request:
This is related to https://github.com/openbao/artwork/issues/11.

I have created a clean, new version of the logo that can serve as a source of truth for future adaptions.

pree commented:
I'm unsure if we should change the folder structure, a lot of external integration probably already depend on that. We could replace the existing ones with the `export` ones and add a new `src` directory?

// Edit: I see it's only one source file, I think this can live in the root of the repo, too.

weberval commented:
True. 

IMO we should consider two things here:
1. The old structure does not point out that these files are _not_ meant for further development. I have seen people reusing the wrong file many times.
2. I would consider linking to a file without commit reference bad practice ATM, because we do not have any regulation about the files. If we would like to do that in the future, we should at least constrain the width / height ratio, and maybe the exact dimension for the raster exports.

pree commented:
Not pinning to a specific commit sha results in always having the newest version ^^ But I agree, that we could change the structure, I don't really like the `export` structure that much and would more rather point out what the source file is

weberval committed:
feat: technically cleaner version of the logo

I have created a technically cleaner version of the logo and generated
corresponding exports.

I have chosen to keep the structure for now, but I decided to remove the
grayscale version, as I do not see any use case for it.

Signed-off-by: Valentin Weber <valentin.weber@adfinis.com>

weberval committed:
fix: switch to non-text logo variant in readme

Signed-off-by: Valentin Weber <valentin.weber@adfinis.com>

weberval commented:
Instead of creating a new structure, @pree and I decided to stick to the current one for now. 
This way, we separate the discussions about the structure of this repository from the technical rework of the files.

JanMa reviewed (approved):
LGTM, thanks a lot for cleaning this up @weberval and @pree
